test-data/unit/check-protocols.test

```diff
@@ -3138,3 +3138,387 @@ foo(B())  # E: Argument 1 to "foo" has incompatible type "B"; expected "Template
           # N: Following member(s) of "B" have conflicts: \
           # N:     Meta: expected "Type[__main__.Template.Meta]", got "Type[__main__.B.Meta]"
 foo(C())  # OK
+
+[case testProtocolClassObjectAttribute]
+from typing import Protocol
+
+class P(Protocol):
+    foo: int
+
+class B:
+    foo: int
+class C:
+    foo: str
+
+def test(arg: P) -> None: ...
+test(B)  # OK
```

Comment:
Should this require ClassVar?

Reply:
Hm, good question. I think yes. And we can also allow the not annotated (IIRC those are implicitly class variables per PEP 526). Let me update the PR.